packages/nx-plugin/src/schematics/schematic/schematic.spec.ts

```diff
@@ -107,4 +107,29 @@ describe('NxPlugin schematic', () => {
       'my-schematic custom description'
     );
   });
+
+  describe('--unitTestRunner', () => {
+    describe('none', () => {
+      it('should generate files', async () => {
```

Comment:
1 more small fix needed here.

```suggestion
      it('should not generate files', async () => {
```